tests/jerry/number-prototype-to-fixed.js

```diff
@@ -31,14 +31,16 @@ assert((-Infinity).toFixed(0) === "-Infinity");
 assert(NaN.toFixed(0) === "NaN");
 assert((0.0).toFixed(0) === "0");
 assert((0.0).toFixed(1) === "0.0");
-assert((-0.0).toFixed(0) === "-0");
```

Comment:
Note: both Safari and Firefox dev consoles report `(-0.0).toFixed(0) === "-0"` as being `false`